isobus/src/isobus_virtual_terminal_working_set_base.cpp

```diff
@@ -2963,12 +2684,18 @@ namespace isobus
 			// If the first byte is 255, then more bytes are used! 4.6.22.3
 			if (iopData[0] == static_cast<std::uint8_t>(EventID::UseExtendedMacroReference))
```

Comment:
Add an early return with `iopLength < 1` and then this function is robust to not need the check mentioned in the first comment.